frontend/src/reviews/ReviewsList.tsx

```diff
@@ -1,36 +1,55 @@
 import { List } from '@mui/material'
 import { useGetReviewRequestsForUser } from 'actions/review'
-import { useEffect, useState } from 'react'
+import { useGetCurrentUser } from 'actions/user'
+import { useCallback, useEffect, useState } from 'react'
 import EmptyBlob from 'src/common/EmptyBlob'
 import Loading from 'src/common/Loading'
 import MessageAlert from 'src/MessageAlert'
 import ReviewItem from 'src/reviews/ReviewItem'
-import { ReviewRequestInterface } from 'types/interfaces'
+import { Decision, ReviewRequestInterface } from 'types/interfaces'
 
 type ReviewsListProps = {
-  isActive?: boolean
   kind?: 'release' | 'access' | 'all'
 }
 
-export default function ReviewsList({ isActive = true, kind = 'all' }: ReviewsListProps) {
-  const { reviews, isReviewsLoading, isReviewsError } = useGetReviewRequestsForUser(isActive)
+export default function ReviewsList({ kind = 'all' }: ReviewsListProps) {
+  const { reviews, isReviewsLoading, isReviewsError } = useGetReviewRequestsForUser()
+  const { currentUser, isCurrentUserLoading, isCurrentUserError } = useGetCurrentUser()
   const [filteredReviews, setFilteredReviews] = useState<ReviewRequestInterface[]>([])
 
+  const doesNotContainUserResponse = useCallback(
+    (review: ReviewRequestInterface) => {
+      return (
+        currentUser &&
+        review.responses.filter(
```

Comment:
Slight performance improvement if you use a `find` so it exits early if it can:
```
      return (
        currentUser &&
        !review.responses.find(
          (response) => response.user === `user:${currentUser.dn}` && response.decision === Decision.Approve,
        )
      )
```